Blocks/HTTP/impl/posix_server.h

```diff
@@ -75,16 +75,56 @@ struct HandlerDoesNotExistException : current::net::HTTPException {
   using current::net::HTTPException::HTTPException;
 };
 
+struct ServeStaticFilesFromOptions {
+  std::string url_dirname;
+  std::vector<std::string> index_filenames;
+
+  ServeStaticFilesFromOptions(const std::string& url_dirname,
+                              const std::vector<std::string>& index_filenames)
+      : url_dirname(url_dirname), index_filenames(index_filenames) {}
+
+  ServeStaticFilesFromOptions(const std::string& url_dirname)
+      : ServeStaticFilesFromOptions(url_dirname, {"index.html", "index.htm"}) {}
+
+  ServeStaticFilesFromOptions() : ServeStaticFilesFromOptions(std::string{current::url::GetURLPathSeparator()}) {}
```

Comment:
This meas "serve on root by default", right? If yes, URLs always use forward slashes.